### DISCUSSION TITLE
is Band n46 supported currently?

### DISCUSSION THREAD

**Discussion by gakest2000** [srsran/srsRAN_Project#201] [General Help] [answered]
Can srsRAN support TDD band n46?
I cannot set an NR cell on this band, that is the error I have :
![image](https://github.com/srsran/srsRAN_Project/assets/118405043/42082f73-9718-4518-9d4e-48ca63ec995b)
For this cell config:

![image](https://github.com/srsran/srsRAN_Project/assets/118405043/db0abbb8-161f-4661-bf22-0542ee71e25e)

**Comment by ismagom** [comment, marked as answer]
Have you tried setting `common_scs: 30` ?

**Reply by gakest2000** [reply]
Thanks, it works when applying this setting:

![image](https://github.com/srsran/srsRAN_Project/assets/118405043/a6962fa6-38d7-4ee4-9669-d93fe65ea8e3)
